Rakefile

```diff
@@ -14,9 +14,9 @@ FACTER_LIB = File.join(PROJECT_ROOT, 'ruby', 'facter', 'lib')
 PUPPET_SERVER_RUBY_SRC = File.join(PROJECT_ROOT, 'src', 'ruby', 'puppetserver-lib')
 PUPPET_SUBMODULE_PATH = File.join('ruby','puppet')
 # Branch of puppetserver for which to update submodule pins
-PUPPETSERVER_BRANCH = ENV['PUPPETSERVER_BRANCH'] || '5.1.x'
+PUPPETSERVER_BRANCH = ENV['PUPPETSERVER_BRANCH'] || '6.1.0-release'
 # Branch of puppet-agent to track for passing puppet SHA
-PUPPET_AGENT_BRANCH = ENV['PUPPET_AGENT_BRANCH'] || '5.3.x'
+PUPPET_AGENT_BRANCH = ENV['PUPPET_AGENT_BRANCH'] || '6.1.0-release'
```

Comment:
I think these are generally set in ci-job-configs? If you just want to update the default, it seems better to use a more permanent branch?

Reply:
Cool. I looked for a release branch and I don't see one yet.

Reply:
I heard, they're going to be a bit longer, but should have it up by the time the submodule update runs. I'm fine merging this now if you are.